src/components/Icon/Icon.test.js

```diff
@@ -0,0 +1,43 @@
+import { render, screen } from '@testing-library/react';
+import Icon from './index';
+
+describe('Icon', () => {
+  it('The icon container and the icon img are rendered correctly', () => {
```

Comment:
It's a good chances to test default styles when `containerType` is undefined, maybe can add next line:

```
expect(screen.getByRole('img')).toHaveClass('flex justify-center items-center')
```